integration/nwo/token/fabric/opts.go

```diff
@@ -21,3 +26,35 @@ func IsFabricCA(tms *topology.TMS) bool {
 	}
 	return false
 }
+
+func WithEndorsers(tms *topology.TMS, endorsers ...string) *topology.TMS {
+	tms.BackendParams["endorsements"] = true
+	tms.BackendParams["endorsers"] = endorsers
+	return tms
+}
+
+func IsEndorsementEnabled(tms *topology.TMS) bool {
+	v, ok := tms.BackendParams["endorsements"]
+	return ok && v.(bool)
+}
+
+func EndorsersToString(tms *topology.TMS) string {
+	v, ok := tms.BackendParams["endorsers"]
+	if !ok {
+		return ""
+	}
+	endorsersStr := strings.Builder{}
```

Comment:
We could also use strings.Join I guess, but still i suppose using a list is a cleaner option, unless I'm missing something

Reply:
okay